core/controllers/question_editor.py

```diff
@@ -64,14 +64,23 @@ def post(self, comma_separated_skill_ids):
             question = question_domain.Question.from_dict(question_dict)
         except Exception:
             raise self.InvalidInputException
+        skill_difficulties = self.payload.get('skill_difficulties')
+
+        if len(skill_ids) != len(skill_difficulties):
+            raise self.InvalidInputException
```

Comment:
For all the InvalidInputExceptions let's add an error message that says what the issue is.

Reply:
Done